tests/backbeat/tests/api/metrics.js

```diff
@@ -99,66 +144,48 @@ describe.skip('Backbeat replication metrics', function dF() {
 
-                    assert(body.backlog.description);
-                    assert(body.backlog.results);
-                    const resultKeys = Object.keys(body.backlog.results);
-                    assert(resultKeys.includes('count'));
-                    assert(resultKeys.includes('size'));
+        series([
+            next => scalityUtils.putObject(srcBucket, key, Buffer.alloc(10000),
+                (err, res) => {
+                    assert.ifError(err);
+                    versionId = res.VersionId;
                     next();
-                });
-            }),
-    ], done));
-
-    it('should get metrics when specifying valid site', done => series([
-        next => scalityUtils.putObject(srcBucket, key, Buffer.alloc(100), next),
-        next => scalityUtils.compareObjectsAWS(srcBucket, destBucket, key,
-            undefined, next),
-        next => makeGETRequest(
-            `/_/backbeat/api/metrics/crr/${destLocation}`,
-            (err, res) => {
-                assert.ifError(err);
-                assert.equal(res.statusCode, 200);
-                getResponseBody(res, (err, body) => {
+                }),
+            next => scalityUtils.compareObjectsAWS(srcBucket, awsDestBucket, key,
+                undefined, next),
+            next => makeDelayedGETRequest('/_/backbeat/api/metrics/crr/' +
+                `${destAWSLocation}/progress/${awsDestBucket}/` +
+                `${destinationKey}?versionId=${versionId}`,
+                (err, res) => {
                     assert.ifError(err);
-                    const metricTypes = ['backlog', 'completions', 'throughput',
-                        'failures'];
-                    const keys = Object.keys(body);
-                    keys.forEach(key => {
-                        assert(metricTypes.includes(key));
-                        assert(body[key].description);
-                        assert(body[key].results);
-                        const resultKeys = Object.keys(body[key].results);
-                        assert(resultKeys.includes('count'));
-                        assert(resultKeys.includes('size'));
-                    });
+                    assert.equal(res.statusCode, 200);
 
-                    next();
-                });
-            }),
-    ], done));
+                    getResponseBody(res, (err, body) => {
+                        assert.ifError(err);
+                        const keys = Object.keys(body);
+                        assert(keys.includes('description'));
+                        assert(keys.includes('pending'));
+                        assert(keys.includes('completed'));
+                        assert(keys.includes('progress'));
+
+                        next();
+                    });
+                }),
+        ], done);
+    });
 
-    it('should get bucket-level throughput metrics', done => {
+    it('should get object-level throughput metrics', done => {
```

Comment:
This PR https://github.com/scality/Zenko/pull/221 should handle this test case as well. Probably a refactor is in order to move all the backbeat API tests into a single file.

Reply:
for now, i'll remove the object-level tests from this file